app/controllers/users_controller.rb

```diff
@@ -1,18 +1,18 @@
 class UsersController < ApplicationController
-    def new
-        @user = User.new
-    end
+    # def new
+    #     @user = User.new
+    # end
     
-    def create
-        @user = User.new(user_params)
-        if @user.save
-            flash[:notice] = "New user #{@user.email} created"
-            redirect_to root_path and return
-        else
-            flash[:alert] = "Failed to save new user"
-            redirect_to new_user_path and return
-        end
-    end
+    # def create
```

Comment:
you can just delete these too instead of commenting it out and have a comment saying something like "device takes care of create and new"